### PR TITLE
Revisions from previous PR.  Get-GcbqProject

### PR DESCRIPTION
**Pull request by ahandley**
Revisions based on feedback.  Thank you everyone.  

Also relocated for better workflow purposes.

### CODE REVIEW

File: Google.PowerShell/BigQuery/GcbqProject.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    /// <para type="link" uri="(https://cloud.google.com/bigquery/docs/reference/rest/v2/projects)">
+    /// [BigQuery Projects]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "GcbqProject")]
+    public class GetGcbqProject : GcbqCmdlet
+    {
+        protected override void ProcessRecord()
+        {
+            // Create the project list request
```
Comment (quoctruong) [inline Google.PowerShell/BigQuery/GcbqProject.cs]: Nit: Full stop at the end of this comment.

File: Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqProject.Tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿. $PSScriptRoot\..\BigQuery\GcbqCmdlets.ps1
+$project, $zone, $oldActiveConfig, $configName = Set-GCloudConfig
+
+Describe "Get-GcbqProject" {
+
+	It "should return the list of projects that the user has permissions to view" {
+		$batches = Get-GcbqProject
```
Comment (quoctruong) [inline Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqProject.Tests.ps1]: Do we know how to create a project programmatically? If so, then maybe we should create one and test that the result contains that project?

Comment (ILMTitan) [inline Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqProject.Tests.ps1]: I don't think we can. I believe every project needs to be billable to someone. Dynamically creating billable entities seems like a no-go.

Comment (iantalarico) [inline Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqProject.Tests.ps1]: I think we can do this.  Let's discuss offline.

File: Google.PowerShell/BigQuery/GcbqProject.cs (hunk 1 above)
Comment (ahandley) [inline Google.PowerShell/BigQuery/GcbqProject.cs]: Added that in.

**Pull request closed without merging**